docs/diagrams/EditFeatureSequenceDiagram.puml

```diff
@@ -0,0 +1,86 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":EditCommandParser" as EditCommandParser LOGIC_COLOR
+participant "e:EditCommand" as EditCommand LOGIC_COLOR
+participant "result:CommandResult" as CommandResult LOGIC_COLOR
+participant "d:EditPersonDescriptor" as EditPersonDescriptor LOGIC_COLOR
+end box
+
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("edit 1 n/Poppy")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("edit n/Poppy")
+activate AddressBookParser
+
+create EditCommandParser
+AddressBookParser -> EditCommandParser
+activate EditCommandParser
+
+EditCommandParser --> AddressBookParser
+deactivate EditCommandParser
+
+AddressBookParser -> EditCommandParser : parse("n/Poppy")
+activate EditCommandParser
+
+create EditPersonDescriptor
+EditCommandParser --> EditPersonDescriptor : EditPersonDescriptor("n/Poppy")
+activate EditPersonDescriptor
+EditPersonDescriptor --> EditCommandParser : d
+deactivate EditPersonDescriptor
+
+EditCommandParser --> EditPersonDescriptor
```

Comment:
Missing method name. Should be setName?

Reply:
Thanks for informing me, I have made the changes 💯 .